src/constructs/cloudwatch/ec2-alarms.ts

```diff
@@ -0,0 +1,50 @@
+import { ComparisonOperator, MathExpression, TreatMissingData } from "@aws-cdk/aws-cloudwatch";
+import { HttpCodeElb, HttpCodeTarget } from "@aws-cdk/aws-elasticloadbalancingv2";
+import type { GuStack } from "../core";
+import type { AppIdentity } from "../core/identity";
+import type { GuApplicationLoadBalancer } from "../loadbalancing";
+import type { GuAlarmProps } from "./alarm";
+import { GuAlarm } from "./alarm";
+
+/**
+ * Creates an alarm which is triggered whenever the percentage of requests with a 5xx response code exceeds
+ * the specified threshold.
+ */
+export interface Gu5xxPercentageMonitoringProps
+  extends Omit<GuAlarmProps, "evaluationPeriods" | "metric" | "period" | "threshold" | "treatMissingData">,
+    AppIdentity {
+  tolerated5xxPercentage: number;
+  numberOfFiveMinutePeriodsToEvaluate?: number;
+  noMonitoring?: false;
+}
+
+interface GuLoadBalancerAlarmProps extends Gu5xxPercentageMonitoringProps {
```

Comment:
The pattern should be able to pass through the load balancer in addition to the user defined props